functions.php

```diff
@@ -1,22 +1,58 @@
 <?php
-
 function getDB(): PDO
 {
     $db = new PDO('mysql:host=db; dbname=collectionapp', 'root', 'password');
     $db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
     return $db;
 }
 
-function formatRecordsForDisplay(array $record): string
+function extractFromDB(PDO $db): array
 {
-    return 'Artist: ' . $record['Artist'] . '<br>' . 'Title:  ' . $record['Title'] . '<br>Rating:  ' .
-        $record['Rating'] . '<br>Release Date:  ' . $record['Released'] .
-        '<br>Condition:  ' . $record['Condition'] . '<br>' . '<br>';
+    $query = $db->prepare('SELECT `Title`, `Rating`, `Released`, `Condition`, `Artist` FROM `records`;');
+    $query->execute();
+    $records = $query->fetchALL();
+    return $records;
 }
 
-function displayRecords(array $records): string
+function display(array $array): string
+{
+    $result = '';
+    foreach ($array as $display) {
+        if (is_Array($display)) {
+            $result .= '<div class="records"><h2>' . $display['Artist'] . ' - ' . $display['Title'] . '</h2>';
+            $result .= '<p>' . 'Rating: ' . $display['Rating'] . '</p>';
+            $result .= '<p>' . 'Release date: ' . $display['Released'] . '</p>';
+            $result .= '<p>' . 'Condition: ' . $display['Condition'] . '</p><br>';
+            $result .= '<br><br><br><br></div>';
+
+        } else {
+            return 'Something went wrong';
+        }
+    }
+    return $result;
+}
+function applyToDB()
 {
-    foreach ($records as $record) {
-        echo formatRecordsForDisplay($record);
+    $db = getDB();
+    if (!empty($_GET['artist']) && !empty($_GET['title']) && !empty($_GET['rating']) && !empty($_GET['released'])
```

Comment:
What happens if a field is empty? Add an else statement so the user doesn't end up on a blank page.